dcap/types/src/error.rs

```diff
@@ -8,6 +8,45 @@ use mc_sgx_util::{ResultFrom, ResultInto};
 #[cfg(feature = "serde")]
 use serde::{Deserialize, Serialize};
 
+/// Errors interacting with a Quote3
+#[derive(Clone, Debug, Display, Eq, Hash, PartialEq, PartialOrd, Ord)]
+#[non_exhaustive]
+pub enum Quote3Error {
```

Comment:
Renamed this on the move out as the aliasing in `lib.rs` had some interesting side effects.
If one wants to document values they have to use the internal name of `Error`. This results in documentation showing the word `Error` but linking to `Quote3Error`
```shell
warning: unresolved link to `Quote3Error::InputLength`
   --> dcap/types/src/quote3.rs:174:13
    |
174 |     /// * [`Quote3Error::InputLength`] if the length of `bytes` is not large
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no item named `Quote3Error` in scope
    |
    = note: `#[warn(rustdoc::broken_intra_doc_links)]` on by default
```